wit/wavs_worker.wit

```diff
@@ -1,4 +1,4 @@
-package wavs:worker@0.4.0-alpha.2;
+package wavs:worker@0.4.0-alpha.3;
```

Comment:
nit: would be nice to just use alpha.4 to keep everything on the same version

Reply:
agreed that it would be nice, though I vaguely remember a discussion that we're free to let these release labels drift until we settle on stripping the label. I could be wrong about that policy though 😅 

That said - it can get confusing, I think we should make sure everything is aligned before we do the next tag (maybe push everything up to alpha.5 then)...